float32 pitch  # 俯仰角度命令
float32 yaw    # 偏航角度命令
